<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClientWidget</class>
 <widget class="QWidget" name="AuditClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditClientWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBoxInformationClient">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>741</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>671</width>
      <height>196</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButtonOrdinateur">
       <property name="text">
        <string>Nom de l'ordinateur distant</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButtonUtilisateur">
       <property name="text">
        <string>Nom de l'utilisateur</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButtonProcesseur">
       <property name="text">
        <string>Architecture processeur</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButtonOs">
       <property name="text">
        <string>OS de l'ordinateur</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEditOs"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEditProcesseur"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEditUtilisateur"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditOrdinateur"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>971</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="textEditEtat">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>521</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2" colspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4" rowspan="2">
        <widget class="QLineEdit" name="lineEditAdresseIP"/>
       </item>
       <item row="0" column="2" rowspan="2" colspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Num√©ro de port du serveur :</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="lineEditPort"/>
       </item>
       <item row="0" column="0" rowspan="2" colspan="2">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Adresse IPdu serveur :</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonQuitter">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonConnexion">
         <property name="text">
          <string>Connexion</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>120</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat de la connexion</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>112</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Informations client</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>128</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Connexion au serveur</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
